docs/source/settings.rst

```diff
@@ -102,6 +102,8 @@ A string referring to one of the following bundled classes:
 * ``eventlet`` - Requires eventlet >= 0.9.7
 * ``gevent``   - Requires gevent >= 0.13
 * ``tornado``  - Requires tornado >= 0.2
+* ``gthread``  - Requires Python 3
+* ``gaiohttp``  - Requires Python 3 and aiohttp >= 0.21.5
```

Comment:
gthread can actually works with python 2 once the futures are installed. Other than that it's OK :)